Formula/dividat-driver.rb

```diff
@@ -0,0 +1,25 @@
+# setup HOMEBREW_GITHUB_API_TOKEN from https://github.com/settings/tokens with *repo* scope (Full control of private repositories)
```

Comment:
I think this and the use of `GitHubPrivateRepositoryDownloadStrategy` are no longer needed?

Reply:
good catch!